stacks/root.yml

```diff
@@ -103,6 +107,8 @@ Mappings:
       prefix: 15
     Play:
       prefix: 21
+    Id:
```

Comment:
this order seems reasonable, but out of curiosity, how often does ID get hit? publish and prx.org go through it, right? what else?